collections/quickstarts/quickstart_dotnet.md

```diff
@@ -51,32 +52,28 @@ using System.Net.Http.Json;
 using System.Text.Json;
 ```
 
+
 ## Step 3: Create a new Application in your Basis Theory Account
 
 If you don’t already have a Basis Theory account and your first Tenant, create one [here](https://portal.basistheory.com/register).
 
-Within your Basis Theory account, create a new Application. To create a new application, head to our portal [here](https://portal.basistheory.com/applications/create) — this Application’s API Key will enable you to authenticate with the Basis Theory platform and create Tokens within your Tenant.
-
-### 3.1 Enter an Application Name
-This name allows you to identify this application in the future — for this tutorial enter .NET Quickstart”.
-
-<img src="/assets/images/getting_started/application_name_dotnet.png" />
```

Comment:
Do we plan to update these images at all or just remove them?

Reply:
I removed them